deploy/deploy-legion-enclave.Jenkinsfile

```diff
@@ -1,28 +1,17 @@
-def targetBranch = params.GitBranch
```

Comment:
Target branch should not been removed and should be used in `createJenkinsTests` instead of GitBranch (we should specify commit id, not just branch)

Reply:
targetBranch defined inside of a methods

Reply:
Fixed